client.go

```
@@ -96,7 +97,13 @@ func HijackHttpRequest(options HijackHttpOptions) error {
 	clientconn := httputil.NewClientConn(dial, nil)
 	defer clientconn.Close()
 
-	clientconn.Do(req)
```

we ignored all errors here? Dang :(